syntax/markdown.vim

```diff
@@ -83,16 +83,6 @@ syntax region mkdLinkTitle matchgroup=mkdDelimiter start=+"+     end=+"+  contai
 syntax region mkdLinkTitle matchgroup=mkdDelimiter start=+'+     end=+'+  contained
 syntax region mkdLinkTitle matchgroup=mkdDelimiter start=+(+     end=+)+  contained
 
-"HTML headings
```

Comment:
This doesn't feel right, the order should not be that important here.

As an alternative to this can you try adding `mkdCode` to the list of things headings are allowed to contain? Does that fix the problem too?